course_contents/1_intro/notes/03. Comparison Operators.md

```diff
@@ -0,0 +1,47 @@
+Each comparison operator returns True or False
+### Equal `==`
+
+`4 == 4  # True`
+
+`4 == 5 # False`
+
+- Can be used to check the truth value of an operand:
+
+`4 is True  #  True`
+
+`4 is False  # False`
+
+`0 is False  # True`
+
+- Can be used to compare two strings or any other data type:
+
+`"Hello" == "Hello"  # True`
+
+`"Hello" == "HELLO"  # False`
+
+`(100, 50) == 50  # False`
+
+`(100, 50) == (500, 219)  # False`
```

Comment:
I think I would make most of the code blocks here (or all of them?) that are separated into lines, into a single code block.

Reply:
Sure. Changed all of them!